Update the download instructions (#22)
NBISweden/bp-sweden-web: datasets/download/downloading-data.qmd

### Install the sda-cli tool
Follow the guidelines [here](../submission/submission-guide.qmd#install-the-sda-cli-tool) to install the sda-cli tool.

### Generate the public and secret key 
The initial step involves creating a crypt4gh keypair using the sda-cli:

```bash
./sda-cli createKey <keypair_name>
```

where `<keypair_name>` is the base name of the key files. The above command will create two key files named
`keypair_name.pub.pem` and `keypair_name.sec.pem`. The public key (`pub`) will be used alongside with `sda-cli`
and will be used by the system for encryption of the files before downloading, while the private one
(`sec`) will be used by the requester for decrypting the files after downloading.

## Check access

After the user has been granted access to the dataset, the user can check access to the dataset by listing the datasets and their files using the `sda-cli`.

> For listing the datasets that the user has access to in production environment, the user needs to run:

Reviewer: ```suggestion
For listing the datasets that the user has access to, the user needs to run:
```

> For listing the datasets that the user has access to in production environment, the user needs to run:

Reviewer 2: ```suggestion
For listing the datasets that the user has access to in the production environment, the user needs to run:
```

Author: Has been removed from Jocke's suggestion

```bash

> ./sda-cli list -config s3cmd.conf --datasets --url https://download.bp.nbis.se (--bytes)

Reviewer 2: Better to add or point to instructions about what is  `s3cmd.conf` and how it is obtained.

Author: There are instructions about how to obtain it at the beginning of the document

```

> For listing the files of a specific dataset in production environment, the user needs to run:

Reviewer: ```suggestion
For listing the files of a specific dataset, the user needs to run:
```

> For listing the files of a specific dataset in production environment, the user needs to run:

Author: Same here

```bash
./sda-cli list -config s3cmd.conf -dataset <DatasetID> --url https://download.bp.nbis.se (--bytes)
```

> where `<DatasetID>` is the ID of the dataset that the user wants to list the files of. The dataset ID can be found by running the first command.

Reviewer 2: ```suggestion
where `<DatasetID>` is the ID of the dataset for which the user wants to list the files. The dataset ID can be found by running the first command.
```

## Download data

After having acquired access to the datasets, the configuration file and the sda-cli tool, the user can download the encrypted data.

> The user needs to provide the public key that was generated earlier and the configuration file to download the data:

Reviewer 2: ```suggestion
The user needs to provide the public key that was generated earlier, as well as the configuration file, to download the data.
```

```bash
./sda-cli download -config s3cmd.conf -pubkey <public-key-file> -dataset-id <DatasetID> --url https://download.bp.nbis.se -outdir </path/to/output/directory> <filepath_1_to_download> <filepath_2_to_download> ...
```

where:

> - `<public-key-file>` is the public key file that was generated earlier (<keypair_name>.pub.pem)

Reviewer 3: ```suggestion

- `<public-key-file>` is the public key file that was generated earlier (`<keypair_name>.pub.pem`)
```
Added one extra blank line to make the list render properly. Also added ``

> - `<DatasetID>` is the ID of the dataset that the user wants to download the files of

Reviewer 2: ```suggestion
- `<DatasetID>` is the ID of the dataset for which the user wants to download the files
```
